python.tf

```diff
@@ -13,6 +13,11 @@ resource "aws_instance" "python" {
     command = "echo ${aws_instance.python.public_ip} > python/ip"
   }
 
+  # Use the python app IP in node code
+  provisioner "local-exec" {
+    command = "sed -Ei '' \"s/(remoteBaseUri:)[^,]*,/\\1 'http:\\/\\/$(cat python/ip):3000',/\" node/config.js"
```

Comment:
El `''` está por si el comando se ejecuta desde MacOS (En Linux no es necesario, y creo que de todas formas funciona igualmente así en Linux). Ver [esta respuesta de SO](https://stackoverflow.com/questions/16745988/sed-command-with-i-option-in-place-editing-works-fine-on-ubuntu-but-not-mac).

Reply:
Mirá vos, curioso. Buen catch 👍